package.json

```diff
@@ -6,18 +6,18 @@
   "main": "dist/index.js",
   "module": "dist-esm/index.js",
   "scripts": {
-    "prebuild": "rimraf dist/ dist-esm/",
-    "build": "run-p build:cjs build:es",
+    "clean": "rimraf dist/ dist-esm/",
+    "build": "npm run clean && npm run build:cjs && npm run build:es",
```

Comment:
@aminya Any particular reason why you don't want to run it in parallel anymore? It will take much longer this way.

Reply:
npm-run-all is unmaintained. It randomly fails on Windows. That should not be a problem though. For development, we can add a `dev` command which uses TypeScript's watch feature.

Reply:
I added dev command in https://github.com/andywer/threads.js/pull/300/commits/c09017e79ee8bea01ee52617284b6f8d97c48a08

Reply:
Fair enough, but how about [`concurrently`](https://www.npmjs.com/package/concurrently) then?

Reply:
We can try that.